recipes/libftdi/1.x/patches/0001-cmake-targets.patch

```diff
@@ -766,6 +776,62 @@ index f5b7018..03d7b07 100644
          case TYPE_AM:
          case TYPE_BM:
              free_start += 0x14;
+diff --git a/src/ftdi.h b/src/ftdi.h
+index fb45008..d2eed07 100644
+--- a/src/ftdi.h
++++ b/src/ftdi.h
```

Comment:
Is this patch (especially changes related to sources) submitted upstream? We have a policy against patches that are modifying sources... This one IMO is on the safe side: it is adding one macro definition in order to remove some sources, but it is not changing behavior or adding new functionality. I would like to listen to other voices.

Reply:
This patch is essentially the hart of this PR. The sources that are removed will never compile with VS. I'm going submit this patch upstream.

Reply:
I just looked through the liftdi mailing list. There exists a patch which implements the _gettimeofday_ function for msvc. This patch is also working but it adds functionality to the code. I try to support this patch upstream but in the meanwhile it is saver to remove the streaming functionality as proposed.